### PR TITLE
commonlib: added solution to Fingerprinting vulnerability (Issue 8056)

### PR DESCRIPTION
**Pull request by zuhairabid**
## Overview
Added solution for : 'Fingerprinting'

## Related Issues
zaproxy/zaproxy#8056

### CODE REVIEW

**Review by kingthorin — APPROVED** [review]

**Comment by thc202** [comment]
This needs to be rebased (for the changelog update), the add-on was released recently.

**Comment by zuhairabid** [comment]
@thc202 hello can you give me some guide one how to rebase it?

**Comment by thc202** [comment]
From the command line you could do (assuming this repo is `upstream`)
```
git fetch upstream
git rebase upstream/main
```

**Comment by zuhairabid** [comment]
@thc202 I have rebased , Can you check please?

**Comment by thc202** [comment]
You will have to (force) push after that.

File: addOns/commonlib/CHANGELOG.md [hunk 1]
```diff
@@ -12,7 +12,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - A generic UI component for keeping menu items sorted.
 
 ### Changed
-- Add solution to 'Brute Forcing Credit Card Information', 'Content Spoofing', 'Credential and Session Prediction', 'XML Injection' and 'XML External Entities' vulnerabilities (Issue 8056).
+- Add solution to 'Fingerprinting', 'Brute Forcing Credit Card Information', 'Content Spoofing', 'Credential and Session Prediction', 'XML Injection' and 'XML External Entities' vulnerabilities (Issue 8056).
```
Comment (kingthorin) [inline addOns/commonlib/CHANGELOG.md:15]: A new single entry under Unreleased

Comment (zuhairabid) [reply addOns/commonlib/CHANGELOG.md:15]: when I click compare on the force push, I see so many changes like versions, Have I done anything wrong?

Comment (kingthorin) [reply addOns/commonlib/CHANGELOG.md:15]: Seems fine to me. The changelog just needs to be properly updated.

Comment (zuhairabid) [reply addOns/commonlib/CHANGELOG.md:15]: image
Any suggestion ?

Comment (kingthorin) [reply addOns/commonlib/CHANGELOG.md:15]: Yes the addon was released this morning so the only “new” change is this one. Move it to the unreleased section add a new changed heading etc

Amend your commit and force push again.

File: addOns/commonlib/CHANGELOG.md [hunk 2]
```diff
@@ -5,7 +5,7 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## Unreleased
-
+- Add solution to 'Fingerprinting' (Issue 8056).
```
Comment (kingthorin) [inline addOns/commonlib/CHANGELOG.md:8]: > ### Changed
> Added solution to 'F8ngerprinting' (Issue 8056).

**Review by thc202 — APPROVED** [review]

**Comment by thc202** [comment]
Thank you!

**Pull request merged**